src/runner.ts

```diff
@@ -94,7 +113,7 @@ export class ResolveRunner implements Types.IResolveRunner {
     return this._source;
   }
 
-  public async resolve(jsonPointer?: string, opts?: Types.IResolveOpts): Promise<Types.IResolveResult> {
+  public async resolve(opts?: Types.IResolveOpts): Promise<Types.IResolveResult> {
```

Comment:
No sir! Is this required? Should avoid breaking changes in this PR unless we really need them.

Reply:
This isn’t a breaking change, as far as I can tell. The types are really confusing in general. But let’s meet about this.

Reply:
The resolve runner isn't publically exposed, and you can pass the `jsonPointer` on opts, so I just consolidated basically.

Reply:
You're right, it's private - I guess that's fine. Just more noise in the PR is all.